inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/interpolate.cpp

```diff
@@ -0,0 +1,130 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
```

Comment:
Nitpick: C++ standard libraries are usually go last in the include list according to VPU code style.

Reply:
Fixed